server/src/main/resources/resources/generic/tos.html

```diff
@@ -1,4 +1,4 @@
 <h2>Provisioning $ID_NAME credential</h2>
-<p>In the following screens, information will be collected for provisioning of $ID_NAME
-  from $ISSUER_NAME.</p>
+<p>In the following screens, information will be collected to provision <u>$ID_NAME</u>
```

Comment:
why do we want to underline the id name and issuer? maybe `<strong>` is better here?

Reply:
Changing to \<b>. Apparently it doesn't know how to handle \<strong>.